### PR TITLE
Support single building DES templates

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
For the constant and variable DES systems, if there was a single building, then the model would fail to build.

#### What does this PR accomplish?
If the jsonpath returns a dict, then it will assume it is a single building. This is not common, but needed for the aggregated load analysis.

#### How should this be manually tested?
unit tests should pass

#### What are the relevant tickets?
n/a

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ geojson = "^3.1"
 jinja2 = "^3.1.2"
 jsonpath-ng = "~1.6"
 jsonschema = "~4.20"
-modelica-builder = "^0.5.1"
+modelica-builder = "^0.5.2rc3"
```
Comment (nllong) [inline pyproject.toml:39]: we should release 0.5.2 soon-ish

Comment (vtnate) [reply pyproject.toml:39]: I'd go so far as to say that if we merge my open PRs, we should go to 0.6.0. Agreed that should happen soon.

**Review by vtnate — COMMENTED** [review]
The code looks good. I was expecting the test to check behavior when there is only a single building in the sys-param though, since handling that appears to be what the new code supports.

File: tests/system_parameters/test_system_parameters.py [hunk 2]
```diff
@@ -192,6 +192,50 @@ def test_get_param(self):
         value = sp.get_param("not.a.real.path")
         assert value is None
 
+    def test_get_param_with_two(self):
```
[the remark this answers is not in the document]
Comment (vtnate) [reply tests/system_parameters/test_system_parameters.py:195]: Previous test checks a single building. This returns a list, the desired behavior.

**Review by vtnate — APPROVED** [review]

**Pull request merged**